### PR TITLE
First pass at new readme. #5456

### PR DESCRIPTION
**Pull request by pippinsplugins**
#5456

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
...
+* NETbilling
+* BitPay
+* Coinbase
+* Moneybookers / Skrill
+* Payza
+* MercadoPago
+* PagSeguro
 
-**Build Up Your Email Subscribers**
+See our [gateways category](https://easydigitaldownloads.com/downloads/category/gateways/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) in the extensions catalogue for a complete list of supported gateways. There are also a large number of additional gateways developed and supported by [3rd party developers](https://easydigitaldownloads.com/3rd-party-extensions/#gateways).
```
Comment (cklosowski) [inline readme.txt]: the `utm_content` here should be `gateways`, so we can focus on traffic directly going to gateways.

File: readme.txt [hunk 2]
```diff
@@ -132,7 +149,7 @@ Yes, EDD has an extension that allows for downloadable invoices to be generated
 
 Yes, through the use of our commercial addon called [Recurring Payments](https://easydigitaldownloads.com/downloads/recurring-payments/?utm_source=docs&utm_medium=faq_tab&utm_term=recurring_addon&utm_content=faqs&utm_campaign=readme).  [Full documentation here](http://docs.easydigitaldownloads.com/article/1142-recurring-payments---overview?utm_source=docs&utm_medium=faq_tab&utm_term=recurring_addon&utm_content=faqs&utm_campaign=readme)
 
-= Can Users Purchase Products without Using PayPal? =
+= Can customers purchase products without using PayPal? =
```
Comment (cklosowski) [inline readme.txt]: Should we mention Amazon here since it's in core?

File: readme.txt (hunk 1 above)
Comment (pippinsplugins) [inline readme.txt]: Fixing.

File: readme.txt (hunk 2 above)
Comment (pippinsplugins) [inline readme.txt]: Added.

File: readme.txt [hunk 3]
```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
...
-* Multiple files per downloadable product
-* Variable prices for multiple price options per product
-* Bundled products
-* Customizable purchase receipts
-* Earnings and sales charts
-* Detailed purchase and file download logs
-* Data export options for CSV and PDF
-* Add functionality with many [extensions](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme)
-* Developer friendly with dozens of actions and filters. [View developer documentation](http://docs.easydigitaldownloads.com/collection/174-developer-docs?utm_source=docs&utm_medium=description_tab&utm_content=developer_docs&utm_campaign=readme)
+Easy Digital Downloads has been translated into more than two dozen languages and new translation are being added all the time, letting store owners have a natural, native experience, regardless of where you live. If your language is not available, we welcome translation contributions! See our [FAQ](http://docs.easydigitaldownloads.com/article/1082-what-spoken-languages-does-easy-digital-downloads-support?utm_source=docs&utm_medium=description_tab&utm_content=languages&utm_campaign=readme).
```
Comment (brashrebel) [inline readme.txt]: ..regardless of where _they_ live?

File: readme.txt [hunk 4]
```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
...
+* [Software Licensing](https://easydigitaldownloads.com/downloads/software-licensing/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) provides a complete solution for selling software license keys
+* [Recurring Payments](https://easydigitaldownloads.com/downloads/recurring-payments/?utm_source=extension&utm_medium=description_tab&utm_content=recurring-payments&utm_campaign=readme) allows store owners to sell recurring subscriptions for digital products
+* [Frontend Submissions](https://easydigitaldownloads.com/downloads/frontend-submissions/?utm_source=extension&utm_medium=description_tab&utm_content=frontend-submissions&utm_campaign=readme) transforms your store into a full-featured multi-vendor marketplace
+* [Comissions](https://easydigitaldownloads.com/downloads/commissions/?utm_source=extension&utm_medium=description_tab&utm_content=commissions&utm_campaign=readme) enables store owners to easily track earnings for product vendors
```
Comment (brashrebel) [inline readme.txt]: Commissions has two "m"s

Comment (pippinsplugins) [inline readme.txt]: Fixed.

File: readme.txt [hunk 5]
```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
...
+* [Free Downloads](https://easydigitaldownloads.com/downloads/free-downloads/?utm_source=extension&utm_medium=description_tab&utm_content=free-downloads&utm_campaign=readme) creates a better download option for free products
```
Comment (brashrebel) [inline readme.txt]: thinking something more descriptive like "allows free products to be purchased more easily by bypassing the standard checkout process"

File: readme.txt [hunk 6]
```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
...
+* [Zapier](https://easydigitaldownloads.com/downloads/zapier/?utm_source=extension&utm_medium=description_tab&utm_content=zapier&utm_campaign=readme) automates your store by connecting Easy Digital Downloads with more than 700 other web services and products
+* [Reviews](https://easydigitaldownloads.com/downloads/product-reviews/?utm_source=extension&utm_medium=description_tab&utm_content=reviews&utm_campaign=readme) allows customers to leave reviews and feedback on the products they have purchased
+* [Recommended Products](https://easydigitaldownloads.com/downloads/recommended-products/?utm_source=extension&utm_medium=description_tab&utm_content=recommended-products&utm_campaign=readme) increases revenue by showing customers recommendations on additional products to purchase bsased on real sales data
```
Comment (brashrebel) [inline readme.txt]: bsased

File: readme.txt [hunk 7]
```diff
@@ -3,59 +3,80 @@ Author URI: https://easydigitaldownloads.com
...
+* [Dropbox File Store](https://easydigitaldownloads.com/downloads/dropbox-file-store//?utm_source=extension&utm_medium=description_tab&utm_content=dropbox-file-store/&utm_campaign=readme) lets you securely host files in your Dropbox account for more reliable and secure file delivery
 
-Would you like to help translate the plugin into more languages? [Join our WP-Translations Community](http://docs.easydigitaldownloads.com/article/1023-translating-easy-digital-downloads?utm_source=docs&utm_medium=description_tab&utm_content=translating&utm_campaign=readme).
+These are just a few of the extensions we offer to help super-charge your digital store. See our complete [extensions catalogue](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) for more. There are also hundreds of extensions created by Easy Digital Downloads community members, which can be found on our [3rd Party Extensions page](https://easydigitaldownloads.com/3rd-party-extensions//?utm_source=3rd-party-extensions&utm_medium=description_tab&utm_content=3rd-party-extensions&utm_campaign=readme).
```
Comment (brashrebel) [inline readme.txt]: extra `/` after page slug in URL

Comment (pippinsplugins) [inline readme.txt]: Fixed

File: readme.txt (hunk 6 above)
Comment (pippinsplugins) [inline readme.txt]: Fixed.

File: readme.txt (hunk 5 above)
Comment (pippinsplugins) [inline readme.txt]: Updated!

**Pull request merged**